redi/utils/redcapClient.py

```diff
@@ -82,13 +87,29 @@ def send_data_to_redcap(self, data, overwrite=False):
             default REDCap will not overwrite any existing value with a blank.
         :return: response
         :raises RedcapError: if failed to send records for any reason.
+        :If MaxRetryError is caught, the function will try resending the same
+         data for a maximum of MAX_RETRY times before exitting. For each
+         attempt the wait time before sending is (the_attempt_no * 6)
         """
         overwrite_value = 'overwrite' if overwrite else 'normal'
 
         try:
+            # The following line simulates github issue #108:
+            # raise MaxRetryError('', 'localhost:8998', None)
             response = self.project.import_records(data,
-                                                   overwrite=overwrite_value)
+                overwrite=overwrite_value)
             return response
+        except MaxRetryError as e:
+            logger.debug(e.message + ", Attempt no.: " + retry_count)
```

Comment:
This line will generate an error due to concatenating a string and int (retry_count)